docs/spec/api.md

```diff
@@ -606,6 +606,23 @@ Note that the upload url will not be available forever. If the upload uuid is
 unknown to the registry, a `404 Not Found` response will be returned and the
 client must restart the upload process.
 
+### Deleting a Layer
+
+An image may be deleted from the registry via its `name` and `reference`. A
+delete may be issued with the following request format:
+
+    DELETE /v2/<name>/blobs/<reference>
```

Comment:
This is just a digest. A reference is a digest or a tag and blobs are not indexed by tag.